### PR TITLE
feat: display source name along with table

### LINKED ISSUE #6609: Source: display source name along with its table id

### PR DESCRIPTION
**Pull request by tabVersion**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

***PLEASE DO NOT LEAVE THIS EMPTY !!!***

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

as title, provide more info in grafana

- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

## Checklist

~~- [ ] I have written necessary rustdoc comments~~
~~- [ ] I have added necessary unit tests and integration tests~~
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Refer to a related PR or issue link (optional)

resolve #6609

### CODE REVIEW

File: dashboard/package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "framer-motion": "^6.5.1",
     "lodash": "^4.17.21",
     "next": "^12.0.9",
+    "protoc-gen-ts": "^0.8.5",
```
Comment (skyzh) [inline dashboard/package.json:32]: The proto generator should be part of `ts-proto`? Do we really need to add this dependency?

Comment (skyzh) [reply dashboard/package.json:32]: Also if you need to add it, move to devDependency.

**Review by skyzh — APPROVED** [review]

**Comment by skyzh** [comment]
By the way I'd like to double check if this new dep is required. It seems that everything works pretty fine without this dependency even on CI?

File: dashboard/package.json (hunk 1 above)
Comment (tabVersion) [reply dashboard/package.json]: done

**Comment by tabVersion** [comment]
> By the way I'd like to double check if this new dep is required. It seems that everything works pretty fine without this dependency before on CI?

I cannot run `npm run proto-gen` without the dep on my machine

**Pull request merged**